app/partials/reference-search.html

```diff
@@ -0,0 +1,84 @@
+<div>
+    <a href="#/reference/reference-add">Add reference</a>
+    <h2>Reference Term Management</h2>
+    <h3 ng-show="vm.referenceSaved">{{vm.referenceSaved}} has been saved</h3>
+    <h3 ng-show="vm.referenceDeleted">{{vm.referenceDeleted}} has been deleted</h3>
+    <b>Find Reference(s)</b>
+    <div>
+        <div>
+            <span>
+                <span>Find a reference by typing in its name:</span>
+                            <input id="inputNode" type="text" ng-model="vm.query" ng-change="vm.timeoutRefresh()">
+                            <input id="searchButton" name="searchButton" value="Search" type="button" ng-click="vm.refreshResponse()">
+                            <br>
+                            <br>
+
+                <span style="display: inline;">
+                    <div align="center">
+
+                        <span ng-show="vm.isUserTyping && vm.query.length > 0"> <b>{{vm.searchNotification}}</b> </span>
+                        <br>
+                        <img ng-show="vm.isUserTyping && vm.query.length > 0" src="../../moduleResources/webservices/rest/js/swagger-ui/dist/images/inprogress.gif" width="64" height="64">
```

Comment:
Please declare conditions in methods in a controller instead of putting them directly in ng-show.  For example here it would be more readable to see <img ng-show="isQueryRunning()">